share/commands/server-upgrade-techniques

```diff
@@ -130,44 +138,56 @@ while true; do
   case "$1" in
     -u |--upgrade)
       UPGRADE=true
+      ANY_OPTION_DEFINED=true
       shift
       ;;
     -i |--initial)
       INITIAL=true
+      ANY_OPTION_DEFINED=true
       shift
       ;;
     -o |--override)
       OVERRIDE=true
+      ANY_OPTION_DEFINED=true
       shift
       ;;
     -f |--force)
       FORCE=true
+      ANY_OPTION_DEFINED=true
       shift
       ;;
     -c|--commit)
       PARENT_COMMIT="$2"
+      ANY_OPTION_DEFINED=true
       shift 2
       ;;
     -a|--autoupdate-technique-library)
-      # we don't support others option with it
-      do_autoupdate_technique_library
-      break
+      AUTO_UPDATE_TECHNIQUE=true
+      ANY_OPTION_DEFINED=true
+      shift
       ;;
     --set-autoupdate-technique-library)
       SET_OPTION=true
       KEY_OPTION=autoupdate-technique-library
       VALUE_OPTION="$2"
+
       if [ "${VALUE_OPTION}" != "true" ] && [ "${VALUE_OPTION}" != "false" ]
       then
        echo "--set-autoupdate-technique-library requires 'true' or 'false' parameter, ${VALUE_OPTION} given"
        exit 1
       fi
+      ANY_OPTION_DEFINED=true
       shift 2
       ;;
     -s|--show)
+      ANY_OPTION_DEFINED=true
       show_defined_option
       shift
       ;;
+    --during-upgrade)
+      DURING_UPGRADE=true
+      shift
```

Comment:
ANY_OPTION_DEFINED=true ?

Reply:
no, because by itself, it is not sufficient to know what to do